assets/css/app.css

```diff
@@ -1,6 +1,6 @@
 /* This file is for your main application css. */
 
-@import "./phoenix.css";
+/*@import "./phoenix.css";*/
```

Comment:
we may still need this.  I believe it contains the styles for rendering the "loading" when the web socket connection goes down